tests/kani/Intrinsics/SIMD/Operators/div_overflow_fail.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Checks that the `simd_div` and `simd_rem` intrinsics check for overflows.
+// kani-verify-fail
```

Comment:
Do you mind moving this test to the `expected` suite please? I prefer using that suite so you can pin down the exact failure.  `kani-verify-fail` is too broad, and I've seen cases where the test was passing due to an unrelated failure.

I think even if kani-compiler crashes will be considered OK.

Reply:
Absolutely! This makes perfect sense, thanks